src/llmcompressor/modifiers/quantization/calibration.py

```
@@ -30,6 +31,7 @@
     "calibrate_query_hook",
     "calibrate_key_hook",
     "calibrate_value_hook",
+    "get_modules_with_fused_observers",
```

Ruff (0.15.13)

[error] Undefined name `get_modules_with_fused_observers` in `__all__`

(F822)